packages/cli/test/integration-3.test.ts

```diff
@@ -369,7 +371,7 @@ test('list the scopes', async () => {
   expect(exitCode, formatOutput({ stdout, stderr })).toBe(0);
 
   const include = new RegExp(`✔ ${contextName}\\s+${email}`);
-  expect(stdout).toMatch(include);
+  expect(stderr).toMatch(include);
```

Comment:
This is a breaking change since you are changing from printing `stdout` to `stderr`

Reply:
@styfle fixed with e1096b84e